src/components/header/Nav.jsx

```diff
@@ -1,57 +1,77 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import AuthActions from '../../actions/AuthActions';
 
-export default function Nav(props) {
-	let { authenticate, displayName, isLogin, isLogout } = props;
-	return (
-		<nav className="navbar navbar-fixed-top navbar-default ">
-			<div className="container">
-				<div className="navbar-header">
-					<button type="button" className="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
-						<span className="sr-only">Toggle navigation</span>
-						<span className="icon-bar" />
-						<span className="icon-bar" />
-						<span className="icon-bar" />
-					</button>
-					<a className="navbar-brand">NewsFlash</a>
-				</div>
-				<div className="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-					<ul className="nav navbar-nav">
-						{!authenticate ?
-							<li>
-								<NavLink exact activeClassName="active" to="/">
-									Home
-							</NavLink>
-							</li>
-							:
-							<li>
-								<NavLink activeClassName="active" to="/newsfeeds">
-									NewsFeeds
-							</NavLink>
-							</li>
-						}
-					</ul>
-					<ul className="nav navbar-nav navbar-right">
-						{authenticate &&
-							<ul className="nav navbar-nav navbar-right">
+export default class Nav extends React.Component {
```

Comment:
I'm sure you've refactored it [here](https://github.com/iakhator/news-feed-application/blob/develop/src/components/header/Nav.jsx#L16) but i would suggest you using a vs-code jsDoc plugin called `Document this`